doc/usage/extensions/math.rst

```diff
@@ -128,10 +129,12 @@ built:
 
 This extension puts math as-is into the HTML files.  The JavaScript package
 MathJax_ is then loaded and transforms the LaTeX markup to readable math live in
-the browser.
+the browser.  Reminder: you should use the math :rst:dir:`directives <math>` and
+:rst:role:`roles <math>`, not the native MathJax ``$$``, ``\(``, etc.
 
 Because MathJax (and the necessary fonts) is very large, it is not included in
-Sphinx.
+Sphinx and your build will include resources from a external site (by
+default cdnjs.cloudflare.com).
```

Comment:
It would be better to merge this into description of `mathjax_path` setting. What do you think?

Reply:
You are right, it belongs below and is basically already said there.  But then I thought the real reason I change it was that this sentence makes it seem like it won't work by default.  So, I changed this part to just say "it's not included but will work automatically by using a third party site", which I think is useful to have there.  And moved the explicit note warning of third-party resources to the lower part.  It's sort of said twice, but I think it's OK.  What do you think?